src/converters/audio_converter.ml

```diff
@@ -99,3 +99,69 @@ module Samplerate = struct
            in
            f converters_conf#get))
 end
+
+module Channel_layout = struct
+  exception Unsupported
+  exception Invalid_data
+
+  type layout = [ `Mono | `Stereo | `Five_point_one ]
+
+  type converter =
+    layout -> layout -> Frame.audio_t array -> Frame.audio_t array
+
+  type t = {
+    src : layout;
```

Comment:
What do we need this field for?

Reply:
To test if the user is passing incorrect data later:
```ocaml
  let convert { src; converter } input =
    if Array.length input != channels_of_layout src then raise Invalid_data;
    converter input
```